apps/emqx/src/emqx_router.erl

```diff
@@ -435,8 +451,25 @@ has_route_v2(Topic, Dest) ->
     end.
 
 cleanup_routes_v2(Node) ->
-    % NOTE
-    % No point in transaction here because all the operations on filters table are dirty.
+    ?with_fallback(
+        lists:foreach(
```

Comment:
Note: `lists:foreach/2` is probably too silent.. :thinking:  Though we eventually ignore return value of `cleanup_routes/1` in `emqx_router_helper`, it may have sense to return `ok`/`error` instead of `ok`.